WooCommerce/Classes/AppDelegate.swift

```diff
@@ -279,14 +279,23 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 //
 extension AppDelegate {
     func updateSharedConfiguration(_ isPointOfSaleActive: Bool) {
-        // Show/hide app's tab bars
-        guard let tabBarController = AppDelegate.shared.tabBarController else {
+        updateTabBarVisibility(isPointOfSaleActive)
```

Comment:
Another great question and a meme that goes into my Swift folder 😂 

At this point there's no reason to keep this in the AppDelegate, since we're calling it's shared instance and the ServiceLocator's one we can just do this from anywhere. I've moved it to the HubMenu view model as it's where we setup the rest of services that we pass into the point of entry. Happy to update it if you think should go somewhere else instead!

Updated on https://github.com/woocommerce/woocommerce-ios/pull/13785/commits/649a33e56aae583a92e536f9c6c3602a5e9f1745